src/lib/comms/sol-lwm2m.c

```diff
@@ -3008,9 +3008,103 @@ clear_bootstrap_ctx(struct sol_lwm2m_client *client)
     }
 }
 
+static int
+is_authorized(struct sol_lwm2m_client *client,
```

Comment:
On 20-07-2016 15:05, Guilherme Íscaro wrote:
> In src/lib/comms/sol-lwm2m.c
> <https://github.com/solettaproject/soletta/pull/2255#discussion_r71576311>:
>
>> @@ -3008,9 +3008,103 @@ clear_bootstrap_ctx(struct sol_lwm2m_client *client)
>>      }
>>  }
>>
>> +static int
>> +is_authorized(struct sol_lwm2m_client *client,
>
> I guess we should document here that 1 means authorized, 0 means not
> authorized and < 0 mean error.
>

Then, the `is_` prefix won't make much sense, right?

> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/solettaproject/soletta/pull/2255/files/213151aa4ace9a334a2762884b64d67576a2a3e4..34de31699e437fab226dead257b73abb82fd1260#r71576311>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADB8T5Uc58g2vwpuqs9WUS7MBihvPUUFks5qXmNtgaJpZM4JPMIy>.
>